includes/Assets.php

```diff
@@ -449,23 +443,23 @@ public function get_scripts() {
                 'deps' => [ 'jquery' ],
             ],
             'dokan-flot-resize'         => [
-                'src'  => WC()->plugin_url() . '/assets/js/jquery-flot/jquery.flot.resize' . $suffix . '.js',
-                'deps' => [ 'dokan-flot-main' ],
+                'src'     => WC()->plugin_url() . '/assets/js/jquery-flot/jquery.flot.resize.js',
```

Comment:
again, do not remove $suffix packages loaded from wooCommerce, kindly fix all the places